packages/x-adapter-platform/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+    project: 'tsconfig.eslint.json'
+  },
+  overrides: [
+    {
+      files: ['src/**/__tests__/*.types.spec.ts'],
+      rules: {
+        '@typescript-eslint/ban-ts-comment': [
+          'error',
+          {
+            'ts-expect-error': false
+          }
+        ]
+      }
+    }
+  ]
```

Comment:
Why do we need this?

Reply:
This was done in the `x-adapter` so we could use `ts-expect-error` freely while testing the schemas `types`. What I think we should question ourselves is if we are going to do some type testing here too (i.e. validating the `platform` model).

Reply:
maybe we can move this to the root `.eslintrc`?  the `ts-expect-error` can be useful in more places.

I will remove here meanwhile, as it is not necessary